Sources/Workspace/InitPackage.swift

```diff
@@ -15,7 +15,7 @@ import PackageModel
 /// Create an initial template package.
 public final class InitPackage {
     /// The tool version to be used for new packages.
-    public static let newPackageToolsVersion = ToolsVersion(version: "5.3.0")
+    public static let newPackageToolsVersion = ToolsVersion(version: "5.4.0")
```

Comment:
given the other comments can we change this to the "current version", .i.e `SwiftVersion.currentVersion` and make sure w.e have unit tests to make sure it never broken

Reply:
Fixed. Yes, there is already a test that ensures the generated packages are buildable & contain the `// swift-tools-version` comment (`WorkspaceTests/InitTests.swift`)